Automattic/wp-calypso
# HelpCenter: add warning in contact page when user has an active ticket

#### Proposed Changes

* This PR adds a check if the current user has an active support ticket open. If this is the case we display a warning banner on the help center contact page.

| Before  | After |
| ------------- | ------------- |
| <img width="417" alt="Screenshot 2022-07-13 at 16 45 39" src="https://user-images.githubusercontent.com/7000684/178762596-742a7433-fa52-41ea-8e63-7bb5300e6402.png">  | <img width="414" alt="Screenshot 2022-07-13 at 16 44 44" src="https://user-images.githubusercontent.com/7000684/178762698-aa1495a3-2b70-45c3-8226-4e18af2a11d0.png"> |

#### Testing Instructions
- Use this diff D86223-code
 - Checkout this branch
 - In apps/editing-toolkit run `yarn dev --sync`
 - In a sandboxed simple site open the new help center
 - Make sure you have active support ticket, if not open one for testing
 - Click on `Still need help?` and check to see if the banner is displayed.
 - Make sure the banner is not displayed when there are no active support tickets.
 
**Note**
If you don't have an open support ticket, and you open one for testing purposes be patient.
It can take a while for the ticket to be created and returned with the `support-history` response.

#### Pre-merge Checklist

Complete applicable items on this checklist **before** merging into trunk. Inapplicable items can be left unchecked.

Both the PR author and reviewer are responsible for ensuring the checklist is completed.


Related to #64983
